appserver/ejb/ejb-container/src/main/java/com/sun/ejb/EjbInvocation.java

```diff
@@ -282,6 +264,38 @@
     // True if lock is currently held for this invocation
     private boolean holdingSFSBSerializedLock = false;
 
+    private int interceptorIndex;
+
+    public Method beanMethod;
+
+    // Only set for web service invocations.
+    private WebServiceContext webServiceContext;
```

Comment:
WARNING: this can be null. The EjbInvocation contains several overlapping implementations.